app/src/ui/preferences/advanced.tsx

```diff
@@ -114,6 +127,37 @@ export class Advanced extends React.Component<
             onChange={this.onReportingOptOutChanged}
           />
         </div>
+        {(this.state.canUseWindowsSSH || enableExternalCredentialHelper()) && (
+          <h2>Network and credentials</h2>
+        )}
+        {this.renderSSHSettings()}
+        {enableExternalCredentialHelper() && (
+          <div className="advanced-section">
+            <Checkbox
+              label={'Use Git Credential Manager'}
+              value={
+                this.state.useExternalCredentialHelper
+                  ? CheckboxValue.On
+                  : CheckboxValue.Off
+              }
+              onChange={this.onUseExternalCredentialHelperChanged}
+              ariaDescribedBy="use-external-credential-helper-description"
+            />
+            <div
+              id="use-external-credential-helper-description"
+              className="git-settings-description"
+            >
+              <p>
+                Use{' '}
+                <LinkButton uri="https://github.com/git-ecosystem/git-credential-manager">
```

Comment:
We have a `gh.io` short link if you'd prefer to use that, incase this path ever changes in the future.

https://gh.io/gcm

Reply:
:+1: Fixed in 2aa54a9e117ccc7dd86286127d44fbe911359173